### PR TITLE
Split off the upgrade rules of the previous release cycle.

### PR DESCRIPTION
**Pull request by GraionDilach**

### CODE REVIEW

File: OpenRA.Mods.Common/UpdateRules/UpdatePath.cs [hunk 1]
```diff
@@ -64,7 +64,6 @@ public class UpdatePath
 
 			new UpdatePath("release-20180307", new UpdateRule[]
```
Comment (pchote) [inline OpenRA.Mods.Common/UpdateRules/UpdatePath.cs]: This needs to be linked to the next path by inserting `"playtest-20180729",` before the `new UpdateRule[]`.

**Review by pchote — DISMISSED** [review]

**Comment by GraionDilach** [comment]
Fixed everything since.

**Review by IceReaper — DISMISSED** [review]
Looks good to me.

**Review by phrohdoh — DISMISSED** [review]

**Review by abcdefg30 — APPROVED** [review]

**Comment by GraionDilach** [comment]
Oh, thanks. I totally forgot this during the night.

**Pull request merged**